.ci/sparse.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+git clone git://git.kernel.org/pub/scm/devel/sparse/sparse.git
```

Comment:
Don't use git. It takes time. Use tarball instead: http://www.kernel.org/pub/software/devel/sparse/dist/sparse-latest.tar.gz

Reply:
See https://kernelnewbies.org/Sparse/